src/test/java/de/zalando/zmon/dataservice/data/KairosDbStoreTest.java

```diff
@@ -43,6 +44,9 @@
     @Autowired
     private DataServiceMetrics metrics;
 
+    @Autowired
+    private WhitelistedChecks whitelistedChecks;
```

Comment:
Where do you actually verify that the white listing works properly?

Reply:
is it about adding tests or can you please explain what do you mean?